inc/api/deprecated/computer_softwarelicense.class.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+namespace Glpi\Api\Deprecated;
+
+if (!defined('GLPI_ROOT')) {
+   die("Sorry. You can't access this file directly");
+}
+
+/**
+ * @since 9.4.0
```

Comment:
Wrong. Change has been done in 9.4, but the @since tag is used to tell when method has been added (9.5 here)